src/files.ts

```diff
@@ -85,7 +85,8 @@ export async function getProjectFiles(rootDir: string = path.join('.', '/'), cal
       filePaths = filePaths.sort(); // Sort files alphanumerically
       let abortPush = false;
       let nonIgnoredFilePaths: string[] = [];
-      const ignoredFilePaths: string[] = [];
+      let ignoredFilePaths: string[] = [];
+      ignoredFilePaths = ignoredFilePaths.concat(ignorePatterns);
```

Comment:
`ignoredFilePaths` is returned as the list of files that are ignored (to be printed out for the user). Ideally, we want to show the patterns, not all the files.

Reply:
```ts
let ignoredFilePaths: string[] = [];
ignoredFilePaths = ignoredFilePaths.concat(ignorePatterns);
```

===

```ts
let ignoredFilePaths: string[] = [...ignorePatterns];
```